Jenkinsfile.github

```diff
@@ -146,11 +146,11 @@ pipeline {
                     stage('Publish: Image') {
                         steps {
                             script {
-                                publishCsmDockerImage(image: "cray-${env.NAME}", tag: "${image_version}-p${PYTHON_VERSION}", isStable: isStable)
+                                publishCsmDockerImage(image: "${env.NAME}", tag: "${image_version}-p${PYTHON_VERSION}", isStable: isStable)
```

Comment:
@trad511 does renaming this sans-`cray` matter to you?
I like it, but I had left it incase it involved opinions or doc-updates.

Reply:
It seems better to get away from the Cray branding since we are converging on HPE's branding, so a standalone `canu` is more reflective of that.  Even if it stands for CSM, it can easily be re-branded `canu` (canoe)